external-docs/docs/guides/command-line/inspect-command.md

```diff
@@ -1,3 +1,939 @@
-# Inspect Command
+# inspect command
+The inspect command is used to inspect the metadata of an existing Qube.
 
-todo: Complete me!
\ No newline at end of file
+**Syntax**  
+`csvcubed inspect [OPTIONS] TIDY_CSV_PATH`
```

Comment:
Can you create a ticket so we can split this information up a bit? It's very verbose at the moment and is likely to be confusing to the user.

Perhaps by default just the catalogue metadata comes out but the user should be able to request `structure` and `data` as well.

Maybe `structure` shows the DSD in cubes and the hierarchy (or a portion of it, if it is too large) in code-lists.

The `data` should show the first *or* last 10 observations IMO, or just a random sample of 10 from somewhere in the data set. It should also show those break-downs by unit + measure.

Reply:
And maybe there should just be an `all` option to show everything as well.